### ISSUE TITLE
model initialization

### ISSUE THREAD

**Issue by Faye0123** [bcmi/CaGNet-Zero-Shot-Semantic-Segmentation#15]
Hi thanks for the code!

I have a question concerning the model initialization for training on cocostuff dataset.
In the configs/cocostuff.yaml file, seems you are using the SPNet trained model as initialization. Is that fair?

![1](https://user-images.githubusercontent.com/69186641/115338433-67696b00-a1d5-11eb-9920-ac28ba782ee4.png)

**Comment by Siyuan-Zhou** [comment]
@Faye0123 Thanks for your question. For the cocostuff dataset, we use part of the SPNet trained model as initialization for the backbone network of the generator, because our deeplabv2 backbone is almost the same with SPNet. This reduces the training time a lot when the dataset is difficult. Actually, we have tried to initialize with only imagenet pretrained weight for cocostuff dataset, in which case you have to additionally train the backbone for several epochs at first, with the remaining steps unchanged. The final evaluation results on test set are similar to our reported ones. Therefore, you may choose to either directly use SPNet trained weight or train the backbone on your own for several epochs at first.
